activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   Added PostgreSQL migration commands for enum drop, rename, add value, and
+    rename value.
```

Comment:
Would you add some usage for each commands here?

Reply:
This has been added ❤️